### ISSUE TITLE
Resuming playback after waking from sleep

### ISSUE THREAD

**Issue by InfiniteImp** [ZeBobo5/Vlc.DotNet#607]
I have a question about Vlc.DotNet.

# Generic information

- **Vlc.DotNet version** : 3.1.0.0
- **Vlc.DotNet project used** : WinForms
- **libvlc version** : 3.08.1 via Package Manager
- **.net version** : 4.7.2
- **Project language** : VB.net
- **Project build architecture** : AnyCPU 
- **Operating system** : Windows 10 x64 1909

# Summary

I have a scenario where I start playing a movie (from the network), pause it, then leave the player idle.  Eventually the computer goes to sleep.  When I wake it up later and resume from pause, the video plays a second or so (presumably whatever was in the buffer) then playback ends.  I can re-select the video and play it after that, no problem.  I assume this is caused by the network connection taking a few seconds to come back to life after resuming from sleep.  Where can I trap this error and have it retry?  Is there a VLC event that fires if playback is interrupted due to a network error?  Ideally I'd like to add a pause there and have it try to resume.

Can't confirm if this happens in VLC itself as I haven't tried this scenario with it.

**Comment by jeremyVignelles** [comment]
Do you have any verbose logs to share? I'm pretty sure that VLC would behave the same, can you confirm that?

**Comment by InfiniteImp** [comment]
No I don't have any logging that would help here.  I don't have VLC itself installed but I can do that if needed.  For now, I may have figured it out.  I added a FileInfo check (something simple to access the network share) after resuming from my internal screensaver (the screen saver kicks in before the computer goes to sleep so that means it's still in screen saver mode when it wakes up.  So far it's resumed playback properly both times I tested it.  I'll close this item and re-open later if needed.
